src/pages/Login/WelcomeMessage.js

```diff
@@ -45,14 +49,33 @@ const WelcomeMessage = ({ isOnNewEnrollmentPage, targetApi }) => {
               En savoir plus
             </a>
           </p>
+          <p>
+            Pour faire une demande d'accès, allez sur{' '}
+            <a href="https://api.gouv.fr/rechercher-api">api.gouv</a> et faites
+            une demande à l'API de votre choix.
+          </p>
```

Comment:
I do not get what problem is solved by this. To me this seems not useful in most of the cases.

case 1 : user coming from api.gouv.fr might click on this and make a loop that push him back where he came from.
case 2 : user coming from email notification (ex : « your demand as been validated », or « you are the dpd contact »), might not find this link useful.
case 3 : people coming directly to https://datapass.api.gouv.fr might find this link useful but this kind of access must be from instructors mostly.

I suggest to rather improve or replace the previous <p> to keep this page simple.

Reply:
Plutôt pas d'accord mais ok pour supprimer comme ca on déploie